docs/reference/tctl/activity/complete.md

```diff
@@ -67,10 +61,15 @@ tctl activity complete --result <value>
 
 ### `--identity`
 
-How to specify the identity of the operator when using tctl to complete an <preview page={WhatIsAnActivityExecution}>Activity Execution</preview>.
+How to specify the identity of the operator when using tctl to complete an [Activity Execution](/docs/content/what-is-an-activity-execution).
```

Comment:
I think we are going to need more info here eventually. I don't know what the `--identify` modifier is

Reply:
Agreed.